### PR TITLE
PR:  Algo Bubble Sort  in JavaScript

### PR DESCRIPTION
**Pull request by sakshipatil4403**
Description:
This pull request introduces the Bubble Sort algorithm implemented in JavaScript, along with a comprehensive README file.

Changes Made:
Added JavaScript code for the Bubble Sort algorithm.
Included a README file with the following sections:
Definition: Describes the algorithm's purpose and operation.
Explanation in Simple Terms: Provides a non-technical explanation for easy understanding.
Code-Related Explanation: Offers a detailed breakdown of the JavaScript code.
Included example usage and a sample array for sorting.

Purpose:
The purpose of this contribution is to provide developers with a clear and educational resource for understanding and implementing the Bubble Sort algorithm in JavaScript. The README file ensures that both technical and non-technical users can grasp the concept and its practical application.

 @Abishethvarman    Please review this PR, and if it aligns with the project's goals, consider merging it. Your feedback and suggestions are welcome. Thank you!

### CODE REVIEW

File: Bubble-Sort_Javascript/Bubble-Sort.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function bubbleSort(arr) {
```
Comment (Abishethvarman) [inline Bubble-Sort_Javascript/Bubble-Sort.js:1]: add this code file in the readme file and make a compact one

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by sakshipatil4403** [comment]
Hello @Abishethvarman i made changes in the Readme file

File: Bubble-Sort_Javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+Definition
```
Comment (Abishethvarman) [inline Bubble-Sort_Javascript/README.md:1]: make a h1 heading "Bubble sort in JavaScript"

File: Bubble-Sort_Javascript/README.md
Comment (Abishethvarman) [inline Bubble-Sort_Javascript/README.md:1]: check the file location. file should be in 2-algorithm/2-javacript/and-your-file

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

**Comment by sakshipatil4403** [comment]
Hello @Abishethvarman I kindly request that you review my pull request and, if you find it suitable, consider merging it with the "hacktoberfest-accepted" label. This will not only help me in achieving my Hacktoberfest goals but will also contribute to the growth and development of the project.

**Review by Abishethvarman — APPROVED** [review]
Approved to merge

**Comment by Abishethvarman** [comment]
@Thevapriyan07 please review this PR ASAP and merge it.

**Review by Thevapriyan07 — APPROVED** [review]
Approve to merge

**Pull request merged**